corgi/tasks/errata_tool.py

```diff
@@ -112,18 +127,23 @@ def slow_load_errata(erratum_name, force_process: bool = False):
     logger.info("Finished processing %s", erratum_id)
 
 
-def _get_relation_build_ids(erratum_id: int) -> set[int]:
+def _get_relation_builds(erratum_id: int) -> list[tuple[int, str]]:
     # Get all the PCR with errata_id
-    relation_build_ids = (
+    relation_builds = (
         ProductComponentRelation.objects.filter(
             type=ProductComponentRelation.Type.ERRATA, external_system_id=erratum_id
         )
-        .values_list("build_id", flat=True)
+        .values_list("build_id", "build_type")
         .distinct()
     )
     # Convert them to ints for use in queries and tasks
     # We don't store them as int because another build system not use ints
-    return set(int(b_id) for b_id in relation_build_ids if b_id is not None)
+    builds: list[tuple[int, str]] = []
+    for b_id, build_type in relation_builds:
+        if b_id is None:
+            pass
+        builds.append((int(b_id), str(build_type)))
+    return builds
```

Comment:
I think you meant "continue" here instead of "pass", and it's much more efficient to use a comprehension instead of building the list one-by-one.

I also think that converting the build_type to a string isn't needed - it should already be a string, right? Or maybe this was just to quiet mypy - that's fine if so.

The "if b_id is not None" was just to avoid a mypy bug that got fixed when we upgraded to the latest version. b_id should never be None since the SoftwareBuild.build_id field doesn't have None=True / isn't nullable:
```suggestion
    builds = tuple((int(b_id), build_type) for (b_id, build_type) in relation_builds)
    return builds
```

Reply:
Ok, done.